spark/v3.2/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/analysis/RewriteDeleteFromTable.scala

```diff
@@ -87,22 +87,22 @@ object RewriteDeleteFromTable extends RewriteRowLevelCommand {
     val remainingRowsPlan = Filter(remainingRowsFilter, readRelation)
 
     // build a plan to replace read groups in the table
-    val writeRelation = relation.copy(table = table)
+    val writeRelation = relation.copy(table = operationTable)
```

Comment:
@szehon-ho @rdblue, could you let me know if this is any better? I did not want to make the var name too long to stay within the line limit.